changelogs/unreleased/4353-jxun

```diff
@@ -0,0 +1,5 @@
+Modify generated from schedule's backup name timestamp to UTC timestamp.
+Fix https://github.com/vmware-tanzu/velero/issues/4279
+When backup is created from schedule, and the backup name is not specified, 
+a containing-timestamp generated name will be used. 
+Due to velero client not set timezone to UTC, a local timezone will be used for the generated name.
```

Comment:
```suggestion
Modify generated from schedule's backup name timestamp to UTC timestamp.
```

You don't have to link the issue on the changelog file, it will be auto-generated and linked in the release node.

Reply:
Thanks. Updated according to comment.